src/bin/travis/resources/cwl_conformance_test.wdl

```diff
@@ -0,0 +1,137 @@
+workflow cwl_conformance_test {
+    String cwl_dir
+    String centaur_cwl_runner
+    String conformance_expected_failures
+
+    call get_test_count {
+        input:
+            cwl_dir = cwl_dir,
+            centaur_cwl_runner = centaur_cwl_runner
+    }
+
+    scatter (test_index in range(get_test_count.test_count)) {
+        call run_test_index {
+            input:
+                cwl_dir = cwl_dir,
+                centaur_cwl_runner = centaur_cwl_runner,
+                test_index = test_index
+        }
+    }
+
+    call make_summary {
+        input:
+            test_count = get_test_count.test_count,
+            test_result_codes = run_test_index.test_result_code,
+            conformance_expected_failures = conformance_expected_failures
+    }
+
+    call echo_summary {
+        input:
+            summary_result_text = make_summary.summary_result_text,
+            summary_result_code = make_summary.summary_result_code
+
+    }
+
+    output {
+        String summary_result_text = make_summary.summary_result_text
+        Int summary_result_code = make_summary.summary_result_code
+    }
+}
+
+task get_test_count {
+    String cwl_dir
+    String centaur_cwl_runner
+
+    command {
+        (
+            cd ${cwl_dir}
+            ./run_test.sh RUNNER="${centaur_cwl_runner}" -l | grep '^\[' | wc -l
+        )
```

Comment:
is this subshelling because our command script doesn't protect itself against user commands cd'ing?

Reply:
Whoops, copypasta from `run_test_index` below. `run_test_index` needs to write the exit code into a file back in the original directory. `get_test_count` here does not. Will remove this subshell.